android/app/src/test/java/org/fossasia/openevent/utils/SpeakerImageUriTest.java

```diff
@@ -0,0 +1,40 @@
+package org.fossasia.openevent.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class SpeakerImageUriTest {
+
+    @Test
+    public void shouldReturnNullOnNullOrEmpty() throws Exception {
+        assertNull(Utils.getSpeakerImageUri(""));
+        assertNull(Utils.getSpeakerImageUri(null));
+    }
+
+    @Test
+    public void shouldReturnUrl() throws Exception {
+        String url = "http://website.ext/resource.ext";
+        String https_url = "http://website.ext/resource.ext";
+
+        assertEquals(url, Utils.getSpeakerImageUri(url));
+        assertEquals(https_url, Utils.getSpeakerImageUri(https_url));
+    }
+
+    @Test
+    public void shouldReturnAssetUri() throws Exception {
+        String uri = "/images/speakers/JohnWick.jpg";
+        String sponsors_uri = "/images/sponsors/Google_1.png";
+
+        assertEquals("file:///android_asset/images/speakers/JohnWick.jpg", Utils.getSpeakerImageUri(uri));
+        assertEquals("file:///android_asset/images/sponsors/Google_1.png", Utils.getSpeakerImageUri(sponsors_uri));
```

Comment:
You should change method name `getSpeakerImageUri` because method name suggest it is for speaker but actually it is used for both speakers and sponsors